packages/react-paypal-js/lint-staged.config.js

```diff
@@ -1,13 +1,13 @@
 module.exports = {
     "**/*.ts?(x)": (filenames) => [
         "npm run typecheck",
-        `prettier --write ${filenames.join(" ")}`,
+        // `prettier --write ${filenames.join(" ")}`,
```

Comment:
What's the reason for omitting these?

Reply:
Since i removed the prettier dependency in react-paypal-js, I thought we have to remove it. I was trying it out while fixing the build errors. Should i put it back?

Reply:
we should actually remove this whole file, all of the `lint-staged` stuff should be handled by the root of the project